index.html

```diff
@@ -294,26 +294,26 @@ <h1 class="about-text">About Myself</h1>
         </div>
         </section>
 
-        <!-- Contact form Section -->
-        <section class="contact-section">
-            <div class="contact-form">
-                <h6>Contact me</h6>
-                <p class="contact-me-text">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+   <!-- Contact form Section -->
+   <section class="contact-section">
+    <div class="contact-form">
+        <h6>Contact me</h6>
+        <p class="contact-me-text">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
 
-                <form name="simple-contact-form" accept-charset="utf-8" action="https://formspree.io/f/moqbpdge " method="post">
-                    <fieldset>
-                    <label for="full-name"></label>
-                    <input class="name-input" type="text" name="name" id="full-name" placeholder="Yeremias" maxlength="30" required="">
-                    <label for="email-address"></label>
-                    <input class="email-input" type="email" name="_replyto" id="email-address" placeholder="natajayanj14@gmail.com" required="">
-                    <label for="message"></label>
-                    <textarea class="body-input" rows="5" name="message" id="message" maxlength="500" placeholder="Write your message here." required=""></textarea>
-                    <input type="hidden" name="_subject" id="email-subject" value="Contact Form Submission">
-                    </fieldset>
-                    <input class="form-btn-submit" type="submit" value="Get it Touch">
-                </form>
-            </div>
-        </section>
+        <form name="simple-contact-form" accept-charset="utf-8" action="https://formspree.io/f/moqbpdge " method="post">
+            <fieldset>
+            <label for="full-name"></label>
+            <input class="name-input" type="text" name="name" id="full-name" placeholder="Yeremias" maxlength="30" required="">
+            <label for="email-address"></label>
+            <input class="email-input" type="email" name="_replyto" id="email-address" placeholder="natajayanj14@gmail.com" required="">
+            <label for="message"></label>
+            <textarea class="body-input" rows="5" name="message" id="message" maxlength="500" placeholder="Write your message here." required=""></textarea>
+            <input type="hidden" name="_subject" id="email-subject" value="Contact Form Submission">
+            </fieldset>
+            <input class="form-btn-submit" type="submit" value="Get it Touch">
+        </form>
```

Comment:
- [OPTIONAL] It is not necessary to assign any value to the `required` attribute, you can only write `required` and it still works correctly. I highly recommend you change that. :wink: